manifests/drupalconsole.pp

```diff
@@ -0,0 +1,22 @@
+class forumone::drupalconsole ($url = 'https://drupalconsole.com/installer')
+{
+  exec { 'forumone::drupalconsole::download':
+    command => "wget ${url} -O /usr/local/bin/drupal",
+    path    => '/usr/bin',
+    creates => '/usr/local/bin/drupal',
+    }
+
+  exec { 'forumone::drupalconsole::drupalinit':
```

Comment:
Should this be run as `root` or as the `vagrant` user? You might need to look at the outcome of this command to see what files it creates.